### ISSUE TITLE
Bug in _undefineValidationScope()

### ISSUE THREAD

**Issue by swift1** [kazupon/vue-validator#38]
Description of the problem:
https://github.com/vuejs/Discussion/issues/288

Example:
https://jsfiddle.net/q7xcbuxd/19/

Solution:

``` javascript
      beforeDestroy: function () {
        this._undefineProperties()
       // this._undefineValidationScope()
      }
```

When commenting out `this._undefineValidationScope()` in vue-validator.js (line 390), everything works as normal.

**Comment by kazupon** [comment]
Thanks for your issue report.

I understand the cause.
This problem is caused by plugin bad design.
I'll fix by temporary way for now.
In next version 2.0, I'll try to implement full-scratch by new design (see #24)

**Comment by swift1** [comment]
Thanks!

**Comment by kazupon** [comment]
I fixed and release v1.3.3.
Please try it. :smiley:

**Comment by swift1** [comment]
It works now. Thanks! :)
